Tests/Common/Util/CurrencyPairUtilTests.cs

```diff
@@ -105,6 +105,26 @@ public void ComparePairWorksCorrectly()
             Assert.AreEqual(CurrencyPairUtil.Match.NoMatch, ethusd.ComparePair("BTC", "USD"));
         }
 
+        [Test]
+        public void ComparePairWorksCorrectlyWithStableCoins()
+        {
+            var btcusdt = Symbol.Create("BTCUSDT", SecurityType.Crypto, Market.Binance);
+            var usdtbtc = Symbol.Create("USDTBTC", SecurityType.Crypto, Market.Binance);
+            var btcusd = Symbol.Create("BTCUSD", SecurityType.Crypto, Market.Binance);
+
+            Assert.AreEqual(CurrencyPairUtil.Match.ExactMatch, btcusdt.ComparePair("BTC", "USD"));
+            Assert.AreEqual(CurrencyPairUtil.Match.InverseMatch, usdtbtc.ComparePair("USD", "BTC"));
+            Assert.AreEqual(CurrencyPairUtil.Match.NoMatch, btcusd.ComparePair("BTC", "EUR"));
```

Comment:
Nice! All these should actually be test cases of a 2 line unit test
For `USDTBTC` `usdtbtc.ComparePair("USD", "BTC")` shouldn't be inverse match but exact match instead? Think this is a bug because of the way symbol is decomposed, might be fixed if using `Decompose` method